--- project.mk	2022-04-03 09:02:47.265248300 -0500
+++ project-new.mk	2022-04-03 15:05:05.244196200 -0500
@@ -183,21 +183,6 @@
 include $(BL60X_SDK_PATH)/make_scripts_riscv/common.mk
 include $(BL60X_SDK_PATH)/make_scripts_riscv/toolchain.mk
 
-# In this recipe, "pip install" may need to build native code, so make sure it
-# doesn't use the cross-compilation environment. The specific list of variables
-# unset is taken from the intersection of what we export and what Python's
-# distutils uses (see CPython's Lib/distutils/sysconfig.py).
-all:
-ifeq ("$(OS)","Windows_NT")
-else
-ifeq ("$(CONFIG_CHIP_NAME)", "BL602")
-	@env -u CC -u CXX -u AR -u CFLAGS -u CPPFLAGS python3 -m pip install -r $(BL60X_SDK_PATH)/image_conf/requirements.txt
-	@cd $(BL60X_SDK_PATH)/image_conf && python3 flash_build.py $(PROJECT_NAME) $(CONFIG_CHIP_NAME)
-endif
-endif
-	@echo "Building Finish. To flash build output."
-
-
 # If we have `version.txt` then prefer that for extracting BL60x_SP_SDK version
 ifeq ("$(wildcard ${BL60X_SDK_PATH}/version.txt)","")
 BL_SDK_VER := $(shell cd ${BL60X_SDK_PATH} && git describe --always --tags --dirty)
